Week2/Section2_w2/MySpotify/Playlist.cpp

```diff
@@ -0,0 +1,60 @@
+#include <iostream>
+#include "Playlist.h"
+#include "Song.h"
+
+
+Playlist::Playlist()
+{
+	PlaylistTitle = "";
+	SongCount = 0;
+}
+
+Playlist::Playlist(std::string NewTitle)
+{
+	PlaylistTitle = NewTitle;
+	SongCount = 0;
+}
+
+bool Playlist::AddSong(Song NewSong)
+{
+	if (SongCount < MaxSongCount && NewSong.IsValid()) {
```

Comment:
Careful there with the brace in the same line.

Reply:
upss bad habits